qa/workunits/cephtool/test.sh

```diff
@@ -1597,6 +1597,19 @@ function test_mon_osd()
   ceph osd crush rm foo
   ceph osd dump -f json-pretty | jq ".crush_node_flags" | expect_false grep foo
 
+  # test device class flags
+  osd_0_device_class=$(ceph osd crush get-device-class osd.0)
+  ceph osd add-noup $osd_0_device_class
+  ceph osd add-nodown $osd_0_device_class
+  ceph osd add-noin $osd_0_device_class
+  ceph osd add-noout $osd_0_device_class
```

Comment:
It may be a bit late, but I don't really like this syntax.  I wonder if we should take the opportunity to switch to something more like:

```
ceph osd set-group <flag> <name>
ceph osd unset-group <flag> <name>
```
I don't love the 'group' part much either, tbh.  But we usually do 'set' and 'unset' so it's weird to have 'add' and 'rm' here.  And also weird to put the flag name as part of the command instead of as an argument.